app/src/main/java/app/revanced/integrations/patches/LithoThemePatch.java

```diff
@@ -19,13 +17,22 @@ public class LithoThemePatch {
         -15790321, // comments chip background (new layout)
         -98492127 // video chapters list background
     };
+    private static final int solidBlack = -16777215;
+
+    private static final int trasparent = 0;
```

Comment:
Constants are written in big letters. Please apply the following changes:
```suggestion
    private static final int SOLID_BLACK = -16777215;
    private static final int TRANSPARENT = 0;
```

Also make sure to use `LIGHT` and `DARK` as the constant names for the fields above for consistency.